pcweb/templates/docpage/docpage.py

```diff
@@ -504,7 +504,7 @@ def wrapper(*args, **kwargs) -> rx.Component:
                             overflow="hidden",
                         ),
                         margin_top="120px",
-                        width="15%",
+                        width="15%" if right_sidebar else "none",
```

Comment:
```suggestion
                        width="15%" if right_sidebar else "0",
```

do we want width="none" or display="none"?